amd_openvx/openvx/ago/ago_util.cpp

```diff
@@ -837,6 +913,17 @@ void agoGetDescriptionFromData(AgoContext * acontext, char * desc, AgoData * dat
 		else
 			sprintf(desc + strlen(desc), "array%s:%s," VX_FMT_SIZE "", virt, agoEnum2Name(data->u.arr.itemtype), data->u.arr.capacity);
 	}
+	else if (data->ref.type == VX_TYPE_OBJECT_ARRAY) {
+		/*if(data->u.objarr.itemtype >= VX_TYPE_KHRONOS_OBJECT_START && data->u.objarr.itemtype <= VX_TYPE_KHRONOS_OBJECT_END) {
+			if(data->u.objarr.itemtype != VX_TYPE_DELAY && data->u.objarr.itemtype != VX_TYPE_OBJECT_ARRAY)
+				sprintf(desc + strlen(desc), "objectarray%s:%s," VX_FMT_SIZE "", virt, agoEnum2Name(data->u.objarr.itemtype), data->u.objarr.numitems);
+		}
+		else
+			sprintf(desc + strlen(desc), "objectarray%s:UNSUPPORTED,NULL", virt);*/
+		sprintf(desc + strlen(desc), "objectarray%s:%lu,[", virt, data->u.objarr.numitems);
```

Comment:
Codacy found an issue: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6287409)